commands/commands.go

```diff
@@ -94,435 +94,437 @@ var Commands = []cli.Command{
-						Action:   nil,
-					},
-					{
-						Name:     "map",
-						Hidden:   true,
-						Usage:    "Plot the geographic location of a specific `device` on a mapping service",
-						Category: "location",
-						Action:   nil,
+				{
+					Name:     "location",
+					Usage:    "Get the physical location of a `device` (NOT YET IMPLEMENTED)",
+					Hidden:   true,
+					Category: "assets",
+					Subcommands: []cli.Command{
+						{
+							Name:     "set",
+							Usage:    "Set the geographic location for specific `device`",
+							Category: "location",
+							Action:   nil,
+						},
+						{
+							Name:     "get",
+							Usage:    "Get current geographic location of a specific `device`",
+							Category: "location",
+							Action:   nil,
+						},
+						{
+							Name:     "map",
+							Hidden:   true,
+							Usage:    "Plot the geographic location of a specific `device` on a mapping service",
+							Category: "location",
+							Action:   nil,
+						},
 					},
 				},
-			},
-			{
-				Name:     "find",
-				Usage:    "Blink the LEDs on a specific `device` for 10 seconds to locate it",
-				Hidden:   true,
-				Category: "assets",
-				Action:   nil,
+				{
+					Name:     "find",
+					Usage:    "Blink the LEDs on a specific `device` for 10 seconds to locate it",
+					Hidden:   true,
+					Category: "assets",
+					Action:   nil,
+				},
 			},
 		},
-	},
-	{
-		Name:   "zones",
-		Usage:  "List available zones (NOT YET IMPLEMENTED)",
-		Hidden: true,
-		//Action:, TBD
-	},
-	{
-		Name:   "racks",
-		Usage:  "List available racks within a given `zone` (or all zones if none is specified) (NOT YET IMPLEMENTED)",
-		Hidden: true,
-		//Action:, TBD
-	},
-	{
-		Name:   "health",
-		Usage:  "Check health for a given `zone`, `rack`, or `device` (NOT YET IMPLEMENTED)",
-		Hidden: true,
-		//Action:, TBD
-	},
-	{
-		Name:   "notifications",
-		Usage:  "List notifications for a given `zone`, `rack`, or `device` (NOT YET IMPLEMENTED)",
-		Hidden: true,
-		//Action:, TBD
-		Subcommands: []cli.Command{
-			{
-				Name:  "clear",
-				Usage: "Clear notifications (`all` or `id`)",
-				Flags: []cli.Flag{
-					cli.StringFlag{
-						Name:  "all",
-						Usage: "Clear all notifications",
-					},
-					cli.StringFlag{
-						Name:  "id",
-						Usage: "Clear notifications for a specific `id`",
-						//Destination: &clearNotificationsID,
+		{
+			Name:   "zones",
+			Usage:  "List available zones (NOT YET IMPLEMENTED)",
+			Hidden: true,
+			//Action:, TBD
+		},
+		{
+			Name:   "racks",
+			Usage:  "List available racks within a given `zone` (or all zones if none is specified) (NOT YET IMPLEMENTED)",
+			Hidden: true,
+			//Action:, TBD
+		},
+		{
+			Name:   "health",
+			Usage:  "Check health for a given `zone`, `rack`, or `device` (NOT YET IMPLEMENTED)",
+			Hidden: true,
+			//Action:, TBD
+		},
+		{
+			Name:   "notifications",
+			Usage:  "List notifications for a given `zone`, `rack`, or `device` (NOT YET IMPLEMENTED)",
```

Comment:
Consider making the NYI stuff fail. It would help to ensure that the errors are being handled properly up the line, especially since golang doesn't really throw. (panic I guess)